### ISSUE TITLE
Browser crashes when all values of Bar graph are equal to 1 

### ISSUE THREAD

**Issue by ionutcalara** [chartist-js/chartist#334]
Code  

``` html
 <div class="ct-perfect-fifth ct-chart" data-labels="[&quot;Broccoli&quot;,&quot;Banana&quot;,&quot;Buckwheat&quot;]" data-values="[&quot;1&quot;,&quot;1&quot;,&quot;1&quot;]" id="MostLovedEdible"></div>
```

Javascript

``` javascript
var data9 = {
    labels: jQuery("#MostLovedEdible").data('labels'),
    series: [jQuery("#MostLovedEdible").data('values')]
};
new Chartist.Bar('#MostLovedEdible', data9,options);
```

It may sound stupid , but in some cases when you generate the data dynamically this may happen.

**Comment by nkapatos** [comment]
I am having the same problem, but I've noticed 2 things.

1 - It crashes only when the onlyInteger option has been set to true
2 - It crashes only when the series is filled with 0 or 1. If any element in the series is bigger than 1, everything is fine.

This example crashes unless you change one of the elements to be > 1, or set to false the onlyInteger

```
    new Chartist.Bar('.ct-chart', {
      labels: [1, 2, 3, 4, 5, 6, 7, 8],
      series: [
        [1, 1, 1, 1, 1, 1, 1, 1]
      ]
    }, {
      axisY: {
        onlyInteger: true
      }
    });
```

**Comment by arturm11** [comment]
Reason: Chartist.rho jumps into infinite loop when 'num' = 1

**Comment by arturm11** [comment]
Workaround: set low/high bounds.

**Comment by gionkunz** [comment]
This is a duplicate of #318 and was fixed with 0d66139b6e7b334f120c2e0590c3cf5b1a8882ec
